components/header.jsx

```diff
@@ -32,6 +29,9 @@ var Header = React.createClass({
               <Link to="sessions">Spaces & Sessions</Link>
             </div>
           </div>
+          <div id="tabzilla">
```

Comment:
Ah that's cuz we have too many items there. I'll increase the breakpoint for desktop view from `768px` to `820px` then (unless we wanna shrink the logo and/or shorten the gaps between nav items).